src/main/java/Iris/ToDoList.java

```diff
@@ -0,0 +1,94 @@
+package Iris;
+
+import java.util.ArrayList;
+
+public class ToDoList {
+    private final ArrayList<Task> list;
+
+    public ToDoList(ArrayList<Task> list) {
+        this.list = list;
+    }
+
+    public void add(Task task) {
+        list.add(task);
+    }
+
+    public void remove(int index) {
+        if (index < 1 || index > list.size()) {
+            throw new IndexOutOfBoundsException("Task index is out of range.");
```

Comment:
Should this be removed? since you are removing from an ArrayList, IndexOutOfBoundsException should be automatically thrown in such cases